src/util.h

```diff
@@ -29,6 +29,9 @@
 #include <boost/signals2/signal.hpp>
 #include <boost/thread/exceptions.hpp>
 
+// Application startup time (used for uptime calculation)
+static const int64_t nStartupTime = GetTime();
```

Comment:
This shouldn't be static (a static variable in a header file means that any .cpp file that includes the header will have its own local variable called `nStartupTime`).

Instead, I think you should define this variable in util.cpp and then have an extern declaration here. See ` BITCOIN_CONF_FILENAME` below for example.

I don't know if you just coppied the `DEFAULT_LOG...` constants below, but I believe those should also be defined in util.cpp.